plugins/modules/packaging/os/pulp_repo.py

```diff
@@ -22,6 +22,18 @@
       - Whether or not to add the export distributor to new C(rpm) repositories.
     type: bool
     default: no
+  components:
+    description:
+      - Debian repo components.
+    type: str
```

Comment:
```suggestion
    type: str
    version_added: 2.0.0
```
This needs to be added for all new options.